### PR TITLE
Fix Nuke pipeline and remove some unused sample projects

### PR DESCRIPTION
**Pull request by andrewlock**

### CODE REVIEW

**Review by lucaspimentel — APPROVED** [review]
Thanks!

File: build/_build/Build.Steps.cs [hunk 1]
```diff
@@ -363,30 +363,33 @@ partial class Build
...
+                if (DirectoryExists(newDir))
                 {
-                    Logger.Info($"Skipping '{target}' as already exists");
+                    Logger.Info($"Skipping '{newDir}' as already exists");
                 }
                 else
                 {
-                    CopyDirectoryRecursively(source, target, DirectoryExistsPolicy.Fail, FileExistsPolicy.Fail);
+                    EnsureExistingDirectory(newDir.Parent);
+                    Cmd.Value(arguments: $"cmd /c mklink /J \"{newDir}\" \"{existingDir}\"");
```
Comment (zacharycmontoya) [inline build/_build/Build.Steps.cs:392]: Does the cmd command work on Linux?

**Review by zacharycmontoya — APPROVED** [review]

File: build/_build/Build.Steps.cs (hunk 1 above)
Comment (andrewlock) [reply build/_build/Build.Steps.cs:392]: It doesn't need to, this one is only called in `BuildWindowsIntegrationTests` 🙂

Yes, it kinda sucks that we have completely different builds on each architecture 😉

**Pull request merged**